api/plans/views.py

```diff
@@ -65,6 +66,17 @@ def get_plan_operations_by_params(request):
         fio_patient = f"{i[8]} {i[9][0:1]}.{i[10][0:1]}."
         date_raw = i[3].split('.')
         date_raw = f"{date_raw[2]}-{date_raw[1]}-{date_raw[0]}"
+        update_date = Log.objects.filter(key=i[0], type=80002)
+        create_date = Log.objects.filter(key=i[0], type=80001)
+        tooltip_data = ""
+        for c in create_date:
+            doctor = c.user.get_fio()
+            obj = json.loads(c.body)
```

Comment:
В теории тут может быть невалидный json, не знаю, как в других местах, но тут стоит на это заложиться.
Надо сделать функцию json_safe_parse
Вот заготовка:
```
def json_safe_parse(data: str, fallback):
    try:
        return json.loads(data)
    except:
        return fallback()
```
И потом так
```suggestion
            obj =json_safe_parse(c.body, dict)
```
Потом с проверкой на не пустоту obj, конечно